lib/usd/hdMaya/adapters/materialNetworkConverter.cpp

```diff
@@ -577,7 +577,6 @@ void HdMayaMaterialNetworkConverter::initialize()
                 { HdMayaAdapterTokens->clearcoatRoughness, coatRoughnessConverter },
                 { HdMayaAdapterTokens->opacity, transmissionToOpacity },
                 { HdMayaAdapterTokens->metallic, fixedZeroFloat },
-                { HdMayaAdapterTokens->useSpecularWorkflow, fixedOneInt },
```

Comment:
Should the other things that have useSpecularWorkflow also get removed?

Reply:
Answer is probably yes. The reason they do not result in black is that we systematically do not set a specular color for these networks. If we were to try to set a specular for Blinn and Phong, they would turn black as well.

Reply:
But if it is not broken, maybe we should not fix it?
The UsdPreviewSurface exporters for these nodes set it to 1, so if we decide to change that, we should keep the exporters in sync.